[2023-02-10T23:09:10.860+0200] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T12:35:00+00:00 [queued]>
[2023-02-10T23:09:10.866+0200] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T12:35:00+00:00 [queued]>
[2023-02-10T23:09:10.866+0200] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-02-10T23:09:10.866+0200] {taskinstance.py:1369} INFO - Starting attempt 1 of 4
[2023-02-10T23:09:10.866+0200] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-02-10T23:09:10.872+0200] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): extract_data_from_api> on 2023-01-10 12:35:00+00:00
[2023-02-10T23:09:10.877+0200] {standard_task_runner.py:52} INFO - Started process 17004 to run task
[2023-02-10T23:09:10.880+0200] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'test_process_sales', 'extract_data_from_api', 'scheduled__2023-01-10T12:35:00+00:00', '--job-id', '984', '--raw', '--subdir', 'DAGS_FOLDER/process_sales/mainS.py', '--cfg-path', '/var/folders/1g/4w5x68gx3d92wgygb97jqbd40000gp/T/tmp_o74hm7e', '--error-file', '/var/folders/1g/4w5x68gx3d92wgygb97jqbd40000gp/T/tmpwoyik9e_']
[2023-02-10T23:09:10.882+0200] {standard_task_runner.py:80} INFO - Job 984: Subtask extract_data_from_api
[2023-02-10T23:09:10.910+0200] {task_command.py:371} INFO - Running <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T12:35:00+00:00 [running]> on host admins-MacBook-Air-2.local
[2023-02-10T23:09:10.939+0200] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=admin@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_process_sales
AIRFLOW_CTX_TASK_ID=extract_data_from_api
AIRFLOW_CTX_EXECUTION_DATE=2023-01-10T12:35:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-10T12:35:00+00:00
[2023-02-10T23:09:11.222+0200] {python.py:173} INFO - Done. Returned value was: {'data': {'message': 'Authorization token not set properly. You are not authorised to use this resourse'}}
[2023-02-10T23:09:11.230+0200] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=test_process_sales, task_id=extract_data_from_api, execution_date=20230110T123500, start_date=20230210T210910, end_date=20230210T210911
[2023-02-10T23:09:11.244+0200] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-10T23:09:11.255+0200] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
